src/main/java/frc/robot/RobotContainer.java

```diff
@@ -119,9 +132,12 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    rightBumper.whileTrue(gripperSystem.coneIntake(aLEDSub));
-    rightTrigger.whileTrue(gripperSystem.cubeIntake(aLEDSub));
-    leftTrigger.whileTrue(gripperSystem.outtake(aLEDSub));
+    rightBumper.whileTrue(gripperSystem.coneIntake());
```

Comment:
Even though the functionality of the LED system is being disabled for now, don't change the signature of the command

Reply:
Was fixed/changed back